### PR TITLE
Retranslate_core

### PR DESCRIPTION
**Pull request by Uwonsang**
core문서 전체적으로 번역 및 수정하였습니다.

### CODE REVIEW

File: sources/layers/core.md [hunk 1]
```diff
@@ -1,162 +1,152 @@
...
 
-`Dense`는 다음과 같은 작업을 구현합니다:
-`output = activation(dot(input, kernel) + bias)`
-여기서 `activation`은 `activation` 인수로 전달되는
-성분별 활성화 함수이고, `kernel`은 레이어가 만들어낸
-가중치 행렬이며, `bias`는 레이어가 만들어낸 편향 벡터입니다
-(`use_bias`가 `True`인 경우만 적용 가능합니다).
+`Dense`는 `output = activation(dot(input, kernel) + bias)` 을 실행합니다.
+여기서 `activation`은 `activation` 인자로 전달되는 원소 단위<sub>element-wise</sub> 활성화 함수이고,
+`kernel`은 층이 만들어낸 가중치 행렬<sub>weight matrix</sub>이며,
```
Comment (Naruu) [inline sources/layers/core.md]: 층이 만들어낸 -> 층에서 만들어진
은 어떤가요?

File: sources/layers/core.md [hunk 2]
```diff
@@ -1,162 +1,152 @@
...
+`bias`는 층이 만들어낸 편향<sub>bias</sub> 벡터이며 `use_bias`가 `True`인 경우만 적용 가능합니다.
```
Comment (Naruu) [inline sources/layers/core.md]: `use_bias=True`

File: sources/layers/core.md [hunk 3]
```diff
@@ -1,162 +1,152 @@
...
 
-참고: 레이어의 인풋이 2를 넘는 계수를 갖는 경우
-`kernel`과의 초기 점곱 전에 일렬화됩니다.
+참고: 층의 입력 랭크가 2보다 클 경우, `kernel`과의 내적<sub>dot product</sub>을 하기 전에 층의 차원을 1차원으로 만들어주어야 합니다.
```
Comment (Naruu) [inline sources/layers/core.md]: 층의 차원을 1차원으로 만들어주어야 합니다.
1차원으로 바뀌는 것이 층이 아니라, 층의 입력값이라서
-> 1D 벡터로 형태를 바꿔야합니다.
는 어떤가요?

File: sources/layers/core.md [hunk 4]
```diff
@@ -1,162 +1,152 @@
...
 model = Sequential()
 model.add(Dense(32, input_shape=(16,)))
-# 모델은 (*, 16) 형태의 배열을 인풋으로 받고
+# 모델은 (*, 16) 형태의 배열을 입력으로 받고
 # (*, 32) 형태의 배열을 출력합니다
 
-# 첫 번째 레이어 이후에는,
-# 인풋의 크기를 특정하지 않아도 됩니다:
+# 첫 번째 층 이후에는,
+# 입력의 크기를 특정하지 않아도 됩니다:
```
Comment (Naruu) [inline sources/layers/core.md]: 특정하다는 표현은 잘 사용하지 않는 것 같아서요,
지정하지 않아도/설정하지 않아도
가 더 자연스러울 것 같아요

File: sources/layers/core.md [hunk 5]
```diff
@@ -1,162 +1,152 @@
...
 model.add(Dense(32))
 ```
 
-__인수__
+__인자__
 
-- __units__: 양의 정수, 아웃풋 공간의 차원.
+- __units__: 양의 정수, 출력 공간의 차원.
```
Comment (Naruu) [inline sources/layers/core.md]: 정수 -> `int`

File: sources/layers/core.md [hunk 6]
```diff
@@ -1,162 +1,152 @@
...
-- __activity_regularizer__: 레이어의 아웃풋(레이어의 “활성화”)에
-    적용되는 정규화 함수
-    ([정규화](../regularizers.md)를 참조하십시오).
-- __kernel_constraint__: `kernel` 가중치 행렬에
-    적용되는 제약 함수
-    ([제약](../constraints.md)을 참조하십시오).
+  ([활성화 함수](../activations.md)를 참조하십시오).
+  따로 설정하지 않으면, 활성화 함수가 적용되지 않습니다.
+  (즉, "linear" 활성화 함수: `a(x) = x`).
+- __use_bias__: `Bool`. 층이 편향 벡터를 사용하는지 여부.
```
Comment (Naruu) [inline sources/layers/core.md]: `Bool`->`bool`

File: sources/layers/core.md [hunk 7]
```diff
@@ -1,162 +1,152 @@
...
+- __bias_initializer__: 편향 벡터의 초기값 함수
+  ([초기값 함수](../initializers.md)를 참조하십시오).
+- __kernel_regularizer__: `kernel` 가중치 행렬에 적용되는 규제<sub>regularizer</sub> 함수
+  ([규제 함수](../regularizers.md)를 참조하십시오).
+- __bias_regularizer__: 편향 벡터에 적용되는 규제 함수
+  ([규제 함수](../regularizers.md)를 참조하십시오).
+- __activity_regularizer__: 층의 출력값(층의 “활성화” 값)에 적용되는 규제 함수
+  ([규제 함수](../regularizers.md)를 참조하십시오).
+- __kernel_constraint__: `kernel` 가중치 행렬에 적용되는 제약<sub>constraints</sub> 함수
+  ([제약 함수](../constraints.md)을 참조하십시오).
```
Comment (Naruu) [inline sources/layers/core.md]: constraint는 '제약 함수' 대신 '제약'으로 번역하기로 했던 것으로 기억해요

File: sources/layers/core.md [hunk 8]
```diff
@@ -1,162 +1,152 @@
...
+  ([활성화 함수](../activations.md)를 참조하십시오)
 
-__인풋 형태__
+__입력 형태__
 
-임의의 값. 이 레이어를 모델의 첫 번째 레이어로
-사용하는 경우, 키워드 인수 `input_shape`을 사용하십시오
-(정수 튜플, 샘플 축을 포함하지 않습니다).
+임의의 형태입니다. 모델의 첫 번째 층으로 Activation층을
+사용할 때 키워드 인자 `input_shape`을 함께 사용하여 형태를 지정합니다.
```
Comment (Naruu) [inline sources/layers/core.md]: 키워드 인자 `input_shape`을 함께 사용하여 형태를 지정합니다. 
-> 키워드 인자 `input_shape`로 형태를 지정합니다.
가 더 자연스러울 것 같아요

File: sources/layers/core.md [hunk 9]
```diff
@@ -1,162 +1,152 @@
...
 
-__인풋 형태__
+__입력 형태__
 
-임의의 값. 이 레이어를 모델의 첫 번째 레이어로
-사용하는 경우, 키워드 인수 `input_shape`을 사용하십시오
-(정수 튜플, 샘플 축을 포함하지 않습니다).
+임의의 형태입니다. 모델의 첫 번째 층으로 Activation층을
+사용할 때 키워드 인자 `input_shape`을 함께 사용하여 형태를 지정합니다. 
+`input_shape`는 정수 튜플<sub>tuple of integers</sub>로 샘플 축을 포함하지 않습니다.
```
Comment (Naruu) [inline sources/layers/core.md]: 정수 튜플<sub>tuple of integers</sub>로
-> `int` 튜플
정도로만 표현해도 괜찮을 것 같아요

File: sources/layers/core.md [hunk 10]
```diff
@@ -1,162 +1,152 @@
...
 
-__아웃풋 형태__
+__출력 형태__
 
-인풋과 동일.
+입력 형태와 동일하다.
```
Comment (Naruu) [inline sources/layers/core.md]: -ㅂ니다 체를 사용해주세요
동일하다. -> 동일합니다.

File: sources/layers/core.md [hunk 11]
```diff
@@ -1,162 +1,152 @@
...
-- __rate__: 0과 1사이 부동소수점. 드롭시킬 인풋 유닛의 비율.
-- __noise_shape__: 인풋과 곱할 이진 드롭아웃 마스크의
-    형태를 나타내는 1D 정수 텐서.
-    예를 들어, 인풋이 `(batch_size, timesteps, features)`의
-    형태를 가지는 경우에 드롭아웃 마스크를
-    모든 시간 단계에 대해서 동일하게 적용하고 싶다면
-    `noise_shape=(batch_size, 1, features)`를 사용하면 됩니다.
-- __seed__: 난수 시드로 사용할 파이썬 정수.
+- __rate__: 0과 1사이 `float`. 드롭시킬 입력 유닛의 비율.
+- __noise_shape__: 입력과 곱하게 되는 이진 드롭아웃 마스크<sub>mask</sub>의
```
Comment (Naruu) [inline sources/layers/core.md]: 마스크<sub>mask</sub>
는 소리나는 대로 적은 단어라서, subscript 안적으셔도 될 것 가아요

File: sources/layers/core.md [hunk 12]
```diff
@@ -1,162 +1,152 @@
...
+  형태를 나타내는 1D 정수 텐서.
```
Comment (Naruu) [inline sources/layers/core.md]: 정수 -> `int`

File: sources/layers/core.md [hunk 13]
```diff
@@ -1,162 +1,152 @@
...
 <span style="float:right;">[[source]](https://github.com/keras-team/keras/blob/master/keras/layers/core.py#L462)</span>
+
 ### Flatten
 
 ```python
 keras.layers.Flatten(data_format=None)
 ```
 
-인풋을 일렬화 합니다. 배치 크기에는 영향을 주지 않습니다.
+입력을 1차원으로 펼칩니다. 배치 크기에는 영향을 미치치 않습니다.
```
Comment (Naruu) [inline sources/layers/core.md]: 입력을 1차원으로 펼칩니다.
->입력을 1차원으로 바꿉니다
는 어떨까요?

File: sources/layers/core.md [hunk 14]
```diff
@@ -166,169 +156,171 @@ model.add(Conv2D(64, (3, 3),
...
-- __dtype__: 문자열, 인풋으로 예상되는 데이터 자료형
-    (`float32`, `float64`, `int32`...)
-- __sparse__: 불리언, 만들어 낼 플레이스홀더가 희소한지
-    여부를 특정합니다.
-- __tensor__: `Input` 레이어로 래핑할 선택적 기존 텐서.
-    따로 설정하는 경우, 레이어는 플레이스홀더 텐서를 만들어 내지 않습니다.
+`model = Model(input=[a, b], output=c)`을 실행할 수 있습니다.
+
+추가된 케라스 속성은 다음과 같습니다.
+`_keras_shape`: 케라스의 형태 유추를 통해 전파되는 정수 튜플.
```
Comment (Naruu) [inline sources/layers/core.md]: 정수 -> `int`

File: sources/layers/core.md [hunk 15]
```diff
@@ -1,162 +1,152 @@
 <span style="float:right;">[[source]](https://github.com/keras-team/keras/blob/master/keras/layers/core.py#L796)</span>
+
 ### Dense
 
 ```python
 keras.layers.Dense(units, activation=None, use_bias=True, kernel_initializer='glorot_uniform', bias_initializer='zeros', kernel_regularizer=None, bias_regularizer=None, activity_regularizer=None, kernel_constraint=None, bias_constraint=None)
 ```
 
-보통의 밀집 연결 신경망 레이어.
+일반적인 밀집 연결 신경망 층<sub>densely-connected NN layer</sub>.
```
Comment (Naruu) [inline sources/layers/core.md]: 밀집 연결 -> 완전 연결

File: sources/layers/core.md [hunk 16]
```diff
@@ -1,162 +1,152 @@
...
 
-`Dense`는 다음과 같은 작업을 구현합니다:
-`output = activation(dot(input, kernel) + bias)`
-여기서 `activation`은 `activation` 인수로 전달되는
-성분별 활성화 함수이고, `kernel`은 레이어가 만들어낸
-가중치 행렬이며, `bias`는 레이어가 만들어낸 편향 벡터입니다
-(`use_bias`가 `True`인 경우만 적용 가능합니다).
+`Dense`는 `output = activation(dot(input, kernel) + bias)` 을 실행합니다.
```
Comment (Naruu) [inline sources/layers/core.md]: 👍

**Comment by fuzzythecat** [comment]
감사합니다!
